src/shadowbox/infrastructure/get_port.ts

```diff
@@ -75,7 +75,7 @@ export function isPortUsed(port: number): Promise<boolean> {
       }
       server.close();
     });
-    server.listen({host: 'localhost', port, exclusive: true}, () => {
+    server.listen({port, exclusive: true}, () => {
```

Comment:
Seems like a change of behavior, please double check whether we would like to listen to the port from any hosts.

Reply:
This is intended. The problem I had was that the resolver was listening on localhost:53, with REUSEADDR, so if I check with localhost:53 it succeeds, but if I check with [::]:53 or 0.0.0.0:53 it fails.